plugins/janus_echotest.c

```diff
@@ -325,10 +294,6 @@ void janus_echotest_destroy(void) {
 		g_thread_join(handler_thread);
 		handler_thread = NULL;
 	}
-	if(watchdog != NULL) {
-		g_thread_join(watchdog);
-		watchdog = NULL;
-	}
 
 	/* FIXME We should destroy the sessions cleanly */
```

Comment:
It was an old memo, that is there in master too. We're already cleaning stuff correctly (or we should) here, as destroying the table results in all its values freed with the callback we set when creating the table itself.